include/libtransistor/ipc/time.h

```diff
@@ -7,15 +7,24 @@
 
 #include<libtransistor/types.h>
 
+ipc_object_t time_systemclock_user;
+ipc_object_t time_systemclock_network;
+ipc_object_t time_systemclock_local;
+
```

Comment:
These need to be `extern`, and the `ipc_object_t` type here is not indicative of what the variable actually means. It would be better to have a `system_clock_t` struct with a single `ipc_object_t` member.